assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -58,9 +59,13 @@ const MessageContainer = styled.div`
   border: ${({ theme }) =>
     `1px solid ${rgba(theme.colors.standard.white, 0.24)}`};
   border-radius: ${({ theme }) => theme.borders.radius.medium};
+  z-index: ${({ customZIndex }) => customZIndex || DEFAULT_MESSAGE_Z_INDEX};
```

Comment:
zindex needs to be overridden at the message level so that it can be interacted with